### PR TITLE
Add manufacturer and other product identifiers

### PR DESCRIPTION
**Pull request by edbeeny**
Google snippets warn against brand and gtin numbers missing.
Add manufactures to products (new  section to add manufacture which then links back to products)
EAN, MPN, ISBN, JAN, UPC to products.
Added a new tab on the product page Extra info.

### CODE REVIEW

**Comment by Mesuva** [comment]
Sorry for the delay on this, I've been away - will look at this in the next week

File: src/CommunityStore/Utilities/Installer.php [hunk 1]
```diff
@@ -50,6 +50,7 @@ public static function installSinglePages($pkg, $upgrade = false)
         self::installSinglePage('/dashboard/store/multilingual/products', $pkg);
         self::installSinglePage('/dashboard/store/multilingual/checkout', $pkg);
         self::installSinglePage('/dashboard/store/multilingual/common', $pkg);
+        self::installSinglePage('/dashboard/store/manufactures/', $pkg);
```
[the remark this answers is not in the document]
Comment (Mesuva) [reply src/CommunityStore/Utilities/Installer.php]: Actually, I'm not seeing in the PR a page to add/edit/delete manufacturers - has this been created?

[the remark this answers is not in the document]
Comment (edbeeny) [reply src/CommunityStore/Utilities/Installer.php]: Oh how embarrassing, sorry not sure why these got missed out. Yes a typo in the installer, all fixed.

**Comment by edbeeny** [comment]
Changes requested

File: src/CommunityStore/Utilities/Installer.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mesuva) [reply src/CommunityStore/Utilities/Installer.php]: Nothing to be embarrassed about here, it's a pretty sizeable addition!
It's all about several eyes on the code.

**Comment by Mesuva** [comment]
Haven't forgotten this - I wanted to get out a new release (which I did yesterday), before pulling in this new code for testing.

**Comment by Mesuva** [comment]
I'm in two minds about these additional fields.

While I can see their utility, the Code/SKU field really is what you'd use for an applicable UPC, ISBN, etc, and I'm thinking that if you needed to have multiple fields you'd use product attributes for this, rather than having hard-coded fields. Having _specific_ types of number fields doesn't really sit right with me, as every country may have something specific to them, or even each industry, so picking a handful of common ones still feels a bit hard to actually decide on.

I'm thinking it might be more appropriate to strip this back to keep the manufacturer/brand selector, leaving anything further to an attribute and custom template exercise.

**Comment by edbeeny** [comment]
The only trouble with making it a product attribute, you will then have to add code into the view to make it viewable.
Looking at other eCommerce sites, would it be better to strip it back to only one called Barcode, similar to Shopify?

**Comment by Mesuva** [comment]
I do like the idea of condensing it down to one 'native' field only, and if developers need further fields (which is then going to be quite unlikely) they can use attributes beyond that.

I've merged your code locally - leave the stripping back to just the barcode field to me. I'll then push up to master and get your feedback on that to check you're happy with my tweaks.

**Comment by Mesuva** [comment]
I've included and updated this code as discussed and it's now sitting in master - if you have a chance to review the changes that would be much appreciated.

**Comment by Mesuva** [comment]
An extra note - I've made the manufacturer display of the name and description on the product block controllable via options on the block itself. The barcode I've left off though, as I feel it would be much less likely to want to output this, and if that was the case a custom template would be appropriate to have to use.

**Comment by Mesuva** [comment]
I've extended the editing of manufacturers in the dashboard too, you can link it to a page, and use WYSIWYG for description editing, something to check out

**Pull request merged**